gcloud/streaming/buffered_stream.py

```diff
@@ -20,7 +20,9 @@ def __init__(self, stream, start, size):
         self._stream = stream
         self._start_pos = start
         self._buffer_pos = 0
-        self._buffered_data = self._stream.read(size)
+        self._buffered_data = self._stream.read(size) \
+            if not hasattr(self._stream,
+                           'closed') or not self._stream.closed else ''
```

Comment:
No backslash continuations, please.  This assignment would be easier to read as a normal `if`..`else` statement.